src/Perplex.ContentBlocks/App_Plugins/Perplex.ContentBlocks/styles/perplex.content-blocks-block.less

```diff
@@ -420,15 +422,10 @@
             &__list .slick-arrow:before {
                 background-size: 15% !important;
             }
-
-            &__preview {
-                display: block;
-                right: 4.5rem;
-            }
         }
 
         @media only screen and (min-width: 108em) {
-            width: 20rem;
+            width: 200px;
```

Comment:
Seems to make the layout name div way too narrow. The old `20rem` seems to correspond to `320px` rather than `200px`.

Reply:
@PerplexDaniel I removed the REM's at some places. It was intentional to have a maximum of 200px. 20rem is way larger than intented. We are optimizing for smaller screens, long names should probalbly be avoided, because that will also look dramatically in splitview.

But I agree that we can look into a better solution, probably a little wider or solution involving flexbox.

Will look at it!

Reply:
Hi @PerplexDaniel , 

Made some changes as you suggested:

- Made the layout-name-divs a bit larger on large screens (looked into flexbox, but it would be a bit of a struggle to get them all lined up together)
- Made some padding tighter so more 'title' will be viewed on one rule.
- Improved the way a multi-line title is rendered. It's vertically centerd now, will expand up to 3 lines and will automatically add ellipsis after 3 rules (css only)


_Tested on mobile, and multiple viewport sizes (also small screens) in Chrome, Edge and Firefox._

- screenshots - 
![2021-01-05 17_04_54-Edit_ Test - Content - localhost](https://user-images.githubusercontent.com/13641023/103671380-1e214280-4f7b-11eb-87f8-6ab8e4a353c7.png)

![2021-01-05 17_11_20-Edit_ Home - Content - localhost](https://user-images.githubusercontent.com/13641023/103671172-d8fd1080-4f7a-11eb-896c-6c2f85c3cce7.png)
![2021-01-05 17_09_47-Edit_ Home - Content - localhost](https://user-images.githubusercontent.com/13641023/103671189-ddc1c480-4f7a-11eb-81a8-3a63b7b30984.png)
![2021-01-05 17_05_14-Edit_ Test - Content - localhost](https://user-images.githubusercontent.com/13641023/103671212-e4503c00-4f7a-11eb-8b8e-a3fdc405f2bc.png)

Reply:
Great!